plugins/search/src/components/SearchType/SearchType.stories.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { useState } from 'react';
+import { MemoryRouter } from 'react-router';
+
+import { SearchType } from '../index';
+import { SearchContext } from '../SearchContext';
+
+export default {
+  title: 'Plugins/Search/SearchType',
+  component: SearchType,
+};
+
+const values = ['value-1', 'value-2', 'value-3'];
+
+export const Default = () => {
+  const [types, setTypes] = useState<string[]>([]);
+
+  return (
+    <MemoryRouter>
+      {/* @ts-ignore (value requires more than what is used here) */}
```

Comment:
Prefer casting over `@ts-ignore`, as it's easy to catch too much with that one and ignore other issues by mistake. In this case you could prolly get away with a `value={{ types, setTypes } as any}`, but otherwise you'd want to `as unknown as ...`